src/components/themed/FilledTextInput.tsx

```diff
@@ -651,7 +651,7 @@ const InnerContainer = styled(Animated.View)<{
   ]
 })
 
-const PrefixAnimatedText = styled(Animated.Text)<{
+const PrefixAnimatedText = styled(AnimatedUnscaledText)<{
```

Comment:
I am pretty sure this will chunder for the same reason as the text input wrapper. Can we double-check if that's the case? If so, we'll have to directly apply `allowFontScaling` in each of these places, which would be unfortunate but necessary.

Reply:
This turns out to be fine, since since the animated text component is *inside* the wrapper. This is different from the text inputs, where we animate the *wrapper*. Animating the wrapper is bad, but wrapping the animation is fine. Carry on.